CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ project(GSI)
   option(BUILD_REG_TESTING "Build the Regression Testing Suite" ON)
   option(BUILD_UNIT_TESTING "Build the Unit Testing Suite" OFF)
   option(BUILD_GSDCLOUD_ARW "Build the GSD cloud analysis " OFF)
-  cmake_minimum_required(VERSION 3.16.2)
+  cmake_minimum_required(VERSION 3.16.1)
```

Comment:
This is interesting.   Why do we need to decrease the cmake version when moving to lua format?

Reply:
My bad.  Now I see what is happening.    Comment resolved.